python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -166,8 +166,10 @@ def __call__(self, strs):
             t = self.tokenizer(s, max_length=self.max_seq_length,
                                truncation=True, padding='max_length', return_tensors='pt')
             tokens.append(t['input_ids'])
-            att_masks.append(t['attention_mask'])
-            type_ids.append(t['token_type_ids'])
+            # The masks are small integers. We don't need to use int4 or int8 to store them.
+            # This can signficantly reduce memory consumption.
```

Comment:
Did we measure the memory consumed here? How much are we saving by using these representations? (vs. increasing the chances we run into overflow errors)

Reply:
a lot. Memory consumption reduces by almost 3 times.

Reply:
attention mask is bool. token type ids are small numbers according to the documentation of huggingface.